### PR TITLE
Add ES5 getters to telemetry data

### PR DESCRIPTION
**Pull request by patocallaghan**
While working on https://github.com/ember-codemods/ember-no-implicit-this-codemod/pull/9 and testing against the test app raised with the original issue, [dustinsoftware/ember-quickstart-codemod](https://github.com/dustinsoftware/ember-quickstart-codemod), that not all telemetry data was gathered for the test component, [old-school](https://github.com/dustinsoftware/ember-quickstart-codemod/blob/master/app/components/old-school.js).

```js
import Component from '@ember/component';

export default class OldSchool extends Component.extend({
  // anything which *must* be merged to prototype here
}) {
  constructor() { super(...arguments); this.noSchool = 'No school'; }
  get theOldSchool() {
    return "the old school"
  }
  // normal class body definition here
}
```

The output telemetry looked like:

```json
'ember-quickstart/components/old-school':
   { computedProperties: [],
     ownActions: [],
     ownProperties: [],
     type: 'Component' },
```

With this change the telemetry now returns

```json
'ember-quickstart/components/old-school':
   { computedProperties: [],
     getters: [ 'theOldSchool' ],
     ownActions: [],
     ownProperties: [],
     type: 'Component' },
```

Notice we are also missing out on the `this.noSchool` assignment in the `constructor` but that's a separate issue 😄 

Also I snuck a small `.gitignore` update in too but if you want me to pull that out to a separate PR just let me know.

### CODE REVIEW

File: lib/gather-telemetry.js [hunk 1]
```diff
@@ -109,8 +109,14 @@ async function buildTelemetry(page) {
           }
         });
 
+        const ownDesc = Object.getOwnPropertyDescriptors(source);
+        const getters = Object.keys(ownDesc).filter(
+          key => ownDesc[key].get && !computedProperties.includes(key)
```
Comment (patocallaghan) [inline lib/gather-telemetry.js]: Computed Properties also end up as ES5 getters but we exclude them as they are already stored in `computedProperties`

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,5 @@
 node_modules
 tmp
 *.log
-test/**/yarn.lock
\ No newline at end of file
+test/**/yarn.lock
```
Comment (patocallaghan) [inline .gitignore]: After reading this comment https://github.com/ember-template-lint/ember-template-lint/pull/768#discussion_r307409812 I'm going to remove this

Comment (patocallaghan) [reply .gitignore]: I've updated to remove the .gitignore addition

**Comment by NullVoxPopuli** [comment]
this is lookin' good, but I think part of it is going to need to go into this repo: https://github.com/ember-codemods/ember-codemods-telemetry-helpers/pull/1

**Comment by patocallaghan** [comment]
@NullVoxPopuli I've put moved the telemetry gathering part to https://github.com/ember-codemods/ember-codemods-telemetry-helpers/pull/4

**Comment by NullVoxPopuli** [comment]
telemetry-helper 0.3.0 includes the changes needed for this PR :)

**Comment by patocallaghan** [comment]
Has telemetry-helper 0.3.0 been included in master already?

**Comment by NullVoxPopuli** [comment]
nope

**Comment by patocallaghan** [comment]
Thanks @NullVoxPopuli I've updated the helper package to use 0.3.0 and removed my code

**Comment by NullVoxPopuli** [comment]
Thank you!!!

**Pull request merged**